spec/features/user_can_remove_donation_from_cart_spec.rb

```diff
@@ -0,0 +1,18 @@
+require "rails_helper"
+
+feature "user can remove item from cart" do
+  scenario "they visit their cart" do
+    donation = create(:donation)
+    visit donations_path
+    click_on "Add to Cart"
```

Comment:
That looks good so far, glad to see that we have added the Add to Cart button to donations index.